train.py

```diff
@@ -1006,6 +1014,9 @@ def init_process(args):
 		'--window', default = 'hann_window', choices = ['hann_window', 'hamming_window'], help = 'for frontend'
 	)
 	parser.add_argument('--onnx')
+	parser.add_argument('--validate-onnx', action = 'store_true', help = 'flag is need to set if you validate onnx model with OnnxWrapper without export from pytorch',)
```

Comment:
please follow the convention, rename it into '--onnx-validate'

Reply:
done